README.md

```diff
@@ -61,7 +61,7 @@ The basis of this pattern leverages the foundation provided by the [Multicloud G
 6. Create a local copy of the secret values file that can safely include credentials. Run the following command :
 
     ```shell
-    cp values-secret.yaml.template ~/values-secret-layered-zero-trust.yaml
+    cp values-secret.yaml.template ~/values-secret.yaml
```

Comment:
This should remain as is as it allows for multiple patterns secrets to reside on the local machine

Reply:
Good thinking. Nearly forget about this feature :joy: